svg2tikz/extensions/tikz_export.py

```diff
@@ -1040,16 +1040,15 @@ def _handle_image(self, node):
         
         width =  self.pxToPt(self.unittouu(node.get('width', '0')));
         height = self.pxToPt(self.unittouu(node.get('height', '0')));
-        
+#         opacity = float(dict(map(lambda s : s.split(':'), node.get('style', 'opacity:1.0').rstrip(';').split(';')))['opacity']);
```

Comment:
Is the commit message correct? You comment out opacity

Reply:
We can safely ignore this and yes the commit message is wrong. I will remove this line in the future PRs.